endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints_integration_test.go

```diff
@@ -216,6 +237,53 @@ func Test_AddEndpointData(t *testing.T) {
 		fmt.Sprintf("stored data '%v' does not equal expected store data '%v'", storedEndpt.OrganizationNames, endpt2.OrganizationNames))
 }
 
+func Test_RemoveOldEndpoints(t *testing.T) {
```

Comment:
I think this is a good test but it doesn't actually call `removeOldEndpoints` so it kind of seems like additional test cases to `Test_AddEndpointData` than a test for `removeOldEndpoints`. I think this test should get merged into the `Test_AddEndpointData` and the `Test_RemoveOldEndpoints` should be something like
1. Add an endpoint
2. Add a second endpoint
3. Get the updated_at time for the second endpoint
4. Call removeOldEndpoints using that time and make sure the first endpoint got deleted.

I think it probably makes sense to also check the case that if you have an endpoint with that URL but a different list source and there is an entry in the `fhir_endpoint_info` table with that URL that you don't delete the fhir_endpoint_info entry

Reply:
Merge test and added another one